grapher/downloadTab/DownloadTab.tsx

```diff
@@ -283,6 +301,30 @@ export class DownloadTab extends React.Component<DownloadTabProps> {
                             </div>
                         </button>
                     </div>
+                    {!isEmpty(this.manager.detailRenderers) && (
+                        <div className="static-exports-options">
+                            <input
+                                type="checkbox"
+                                id="shouldIncludeDetailsInStaticExport"
+                                name="shouldIncludeDetailsInStaticExport"
+                                onChange={(): void => {
+                                    this.isReady = false
```

Comment:
this line makes the grapher loading spinner re-appear for 500ms while the images re-render. 

If we don't have it, then instead you see the static export thumbnails flash and change their aspect ratio. I'm not sure which is better.